packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Activity/TransactionDetails/TransactionDetails.tsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Icon } from '@blockchain-com/constellation'
+import { IconArrowLeft } from '@blockchain-com/icons'
+import BigNumber from 'bignumber.js'
+import styled from 'styled-components'
+
+import { EthProcessedTxType } from '@core/transactions/types'
+import { convertGweiToWei } from '@core/utils/eth'
+import { Button, Link, Text, Toast } from 'blockchain-info-components'
+import CryptoAddress from 'components/CryptoAddress/CryptoAddress'
+import CoinDisplay from 'components/Display/CoinDisplay'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { Flex } from 'components/Flex'
+import { Padding } from 'components/Padding'
+
+import TransactionIcon from '../TransactionIcon/TransactionIcon'
+import StatusBadge from './StatusBadge'
+import TransactionDetailsItem from './TransactionDetailsItem'
+
+export interface ITransactionDetailsProps {
+  coin: string
+  onClose(): void
+  status: 'CONFIRMED' | 'PENDING'
+  transaction: EthProcessedTxType
+}
+
+const Wrapper = styled(Flex)`
+  height: 100%;
+  overflow: auto;
+`
+
+const IconWrapper = styled(Flex)`
+  cursor: pointer;
+`
+
+const ButtonWrapper = styled.div`
+  display: grid;
+`
+
+const getTransactionLinkInEthExplorer = (hash: string) =>
+  `https://www.blockchain.com/eth/tx/${hash}`
+
+const TransactionDetails: React.FC<ITransactionDetailsProps> = ({
+  coin,
+  onClose,
+  status,
+  transaction
+}) => {
+  const [toastMessage, setToastMessage] = React.useState('')
+
+  const handleBackClick = () => {
+    onClose()
+  }
+
+  const handleCopyToClipboard = () => {
+    if (!navigator.clipboard) {
+      return
+    }
+    navigator.clipboard.writeText(transaction.hash)
+  }
+
+  return (
+    <Wrapper flexDirection='column'>
+      <Padding bottom={34}>
+        <IconWrapper>
+          <Icon label='go-back' size='md' color='grey600'>
+            <IconArrowLeft onClick={handleBackClick} />
+          </Icon>
+        </IconWrapper>
+      </Padding>
+
+      <Padding bottom={26}>
+        <Text size='20px' lineHeight='30px' color='white' weight={700}>
+          <FormattedMessage
+            id={`plugin.activity.transactionDetails.${transaction.type}`}
+            defaultMessage={transaction.type.toUpperCase()}
+          />{' '}
```

Comment:
does `{' '}`  stand for just a space here? We could  use `&nbsp;`

Reply:
Yes, it's works fine with {' '}